pallets/ajuna-awesome-avatars/src/mock.rs

```diff
@@ -16,29 +16,29 @@
 
 use crate as pallet_ajuna_awesome_avatars;
 use frame_support::traits::{ConstU16, ConstU64};
-use frame_system as system;
+use frame_system::mocking::{MockBlock, MockUncheckedExtrinsic};
 use sp_core::H256;
 use sp_runtime::{
 	testing::Header,
 	traits::{BlakeTwo256, IdentityLookup},
+	BuildStorage,
 };
 
-type UncheckedExtrinsic = frame_system::mocking::MockUncheckedExtrinsic<Test>;
-type Block = frame_system::mocking::MockBlock<Test>;
+type MockAccountId = u32;
 
 // Configure a mock runtime to test the pallet.
 frame_support::construct_runtime!(
 	pub enum Test where
-		Block = Block,
-		NodeBlock = Block,
-		UncheckedExtrinsic = UncheckedExtrinsic,
+		Block = MockBlock<Test>,
+		NodeBlock = MockBlock<Test>,
+		UncheckedExtrinsic = MockUncheckedExtrinsic<Test>,
 	{
-		System: frame_system,
-		TemplateModule: pallet_ajuna_awesome_avatars,
+		System: frame_system::{Pallet, Call, Config, Storage, Event<T>},
+		AjunaAwesomeAvatars: pallet_ajuna_awesome_avatars::{Pallet, Call, Storage, Event<T>},
```

Comment:
I don't think we need to add these
```suggestion
		System: frame_system,
		AjunaAwesomeAvatars: pallet_ajuna_awesome_avatars,
```